llvm/include/llvm/Support/raw_ostream.h

```diff
@@ -526,6 +526,7 @@ class raw_fd_ostream : public raw_pwrite_stream {
   /// this closes the file when the stream is destroyed. If FD is for stdout or
   /// stderr, it will not be closed.
   raw_fd_ostream(int fd, bool shouldClose, bool unbuffered = false,
+                 bool simpleStream = false,
```

Comment:
IMO this parameter should be documented, `simpleStream` is a rather generic name and could apply to a myriad of file-like objects.

Reply:
Yeah, makes sense. I'll fix up documenting the other options as well.